### PR TITLE
chore: changelog 0.21.0

### PR DESCRIPTION
**Pull request by angelozerr**
chore: changelog 0.21.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.21.0](https://github.com/redhat-developer/quarkus-ls/milestone/24?closed=1) (December 3, 2024)
+
+### Enhancements
+
+- Data model template matcher support. See [#1021](https://github.com/redhat-developer/quarkus-ls/pull/1021).
+- Proper fix for comments in TemplateExtension quickfix test. See [#1020](https://github.com/redhat-developer/quarkus-ls/pull/1020).
```
Comment (datho7561) [inline CHANGELOG.md:8]: This is a bug fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.21.0](https://github.com/redhat-developer/quarkus-ls/milestone/24?closed=1) (December 3, 2024)
+
+### Enhancements
+
+- Data model template matcher support. See [#1021](https://github.com/redhat-developer/quarkus-ls/pull/1021).
+- Proper fix for comments in TemplateExtension quickfix test. See [#1020](https://github.com/redhat-developer/quarkus-ls/pull/1020).
+- Add completion support for quarkus.langchain4j.ollama.embedding-model.model-id. See [#1014](https://github.com/redhat-developer/quarkus-ls/issues/1014).
```
Comment (datho7561) [inline CHANGELOG.md:9]: I think this was released in 0.20.0

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.21.0](https://github.com/redhat-developer/quarkus-ls/milestone/24?closed=1) (December 3, 2024)
+
+### Enhancements
+
+- Data model template matcher support. See [#1021](https://github.com/redhat-developer/quarkus-ls/pull/1021).
+- Proper fix for comments in TemplateExtension quickfix test. See [#1020](https://github.com/redhat-developer/quarkus-ls/pull/1020).
+- Add completion support for quarkus.langchain4j.ollama.embedding-model.model-id. See [#1014](https://github.com/redhat-developer/quarkus-ls/issues/1014).
+- Support for multiple Qute template root paths. See [#1005](https://github.com/redhat-developer/quarkus-ls/pull/1005).
```
Comment (datho7561) [inline CHANGELOG.md:10]: I think this was released in 0.20.0

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md:9]: No, 0.20. supports only  `quarkus.langchain4j.ollama.chat-model.model-id`,this issue is to support also  `quarkus.langchain4j.ollama.embedding-model.model-id`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
 
+## [0.21.0](https://github.com/redhat-developer/quarkus-ls/milestone/24?closed=1) (December 3, 2024)
+
+### Enhancements
+
+- Data model template matcher support. See [#1021](https://github.com/redhat-developer/quarkus-ls/pull/1021).
+- Proper fix for comments in TemplateExtension quickfix test. See [#1020](https://github.com/redhat-developer/quarkus-ls/pull/1020).
+- Add completion support for quarkus.langchain4j.ollama.embedding-model.model-id. See [#1014](https://github.com/redhat-developer/quarkus-ls/issues/1014).
+- Support for multiple Qute template root paths. See [#1005](https://github.com/redhat-developer/quarkus-ls/pull/1005).
+- Roq DataMapping support. See [#1017](https://github.com/redhat-developer/quarkus-ls/pull/1017).
```
[the remark this answers is not in the document]
Comment (angelozerr) [reply CHANGELOG.md:12]: No, Roq support which is based on multiple template root paths is a 0.20.0 feature

File: CHANGELOG.md (hunk 3 above)
Comment (angelozerr) [reply CHANGELOG.md:11]: No, multiple template root paths is a 0.21.0 feature

File: CHANGELOG.md (hunk 2 above)
Comment (datho7561) [reply CHANGELOG.md]: my bad, sorry about that

File: CHANGELOG.md (hunk 1 above)
Comment (datho7561) [reply CHANGELOG.md:9]: @angelozerr once this is addressed I think the changelog will be good.

Comment (angelozerr) [reply CHANGELOG.md]: fixed

**Pull request merged**